src/main/daml/Daml/Finance/Interface/Instrument/Bond/Callable/BondTypes.daml

```diff
@@ -0,0 +1,48 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Interface.Instrument.Bond.Callable.BondTypes where
```

Comment:
Any reason for using a separate module and not `Types`?

Reply:
Do you mean `Daml.Finance.Interface.Instrument.Bond.Callable.Types` ?

Reply:
I think I added  additional types there at some point, but got review feedback that it was rather for the instrument type only.

Reply:
Happy to move them though, although I believe these types could be of use to other Bond instruments as well if we choose to update them with these additional features later on.